4.interpret_model/scripts/nbconverted/interpret_model.py

```diff
@@ -40,26 +40,43 @@
 coefs.head()
 
 
-# ### Diagrams for interpreting coefficients
+# ### Save Coefficients Matrix in Tidy Long Format
 
 # In[4]:
 
 
+coefs_save_path = pathlib.Path(f"interpretations/final_model_coefs.tsv")
+coefs_save_path.parents[0].mkdir(parents=True, exist_ok=True)
```

Comment:
Consider using `pathlib.Path.parent` here if able to simplify (removing the need for a `parents` index reference).

Reply:
Can't believe I haven't learned this way of accessing the parent directory before, thanks! Done in [bc1bf11](https://github.com/WayScience/phenotypic_profiling_model/pull/12/commits/bc1bf119e20ff28b38fbd6a0a724834b0ee59941)